run-17495340446-artifacts/scripts/pptx_format.py

```
@@ -0,0 +1,258 @@
+    
+    # Look for title placeholders and apply title formatting
+    for shape in root.iter():
+        if shape.tag.endswith("}sp"):  # Shape element
+            # Check for title indicators in shape properties
+            nvSpPr = shape.find(".//" + P_NS + "nvSpPr")
+            if nvSpPr is not None:
+                nvPr = nvSpPr.find(P_NS + "nvPr")
+                if nvPr is not None:
+                    ph = nvPr.find(P_NS + "ph")
+                    if ph is not None and ph.get("type") in ["title", "ctrTitle"]:
+                        title_found = True
```

Ruff (0.12.2)

Local variable `title_found` is assigned to but never used

Remove assignment to unused variable `title_found`

(F841)